src/models/delegations.ts

```diff
@@ -20,6 +20,7 @@ interface DelegationsOptions {
   unrecognizedFields?: Record<string, JSONValue>;
 }
 
+// A container object storing information about all delegations.
```

Comment:
Just coming at this fresh, I'm still missing quite a bit of context how this is used and how it fits into the overall datamodel. I wonder if this could be included in the comment? Is it that a role has delegations or target have delegations?